linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/ansible/shared.yml

```diff
@@ -0,0 +1,43 @@
+# platform = multi_platform_all
+# reboot = false
```

Comment:
Should `reboot` be true since this is dealing with GRUB?

Reply:
Thanks for noting changed it in 78f7c4112d5f573d0e9327af8465d0b5669696ac